scripts/runBabel.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+/**
+ * This script exists here because you cannot set BABEL_ENV while piping
+ * data to the babel-cli. So this script is able to accept data being piped
+ * in and also set BABEL_ENV. If no data is piped in then it is expected that
+ * the second argument passed to this script will be the src to be run through
+ * babel.
+ */
+
+const babel = require('babel-core');
+const getStdin = require('get-stdin');
+
+// set the environment variable on which babel should run
+process.env.BABEL_ENV = process.env.BABEL_ENV || process.argv[ 2 ];
+
+// check that BABEL_ENV was set
+if(process.env.BABEL_ENV === undefined) {
+  throw new Error('BABEL_ENV is not set. This can be set either by passing in a value to runBabel.js or set via command line');
+}
+
+// read in data piped to this done app
+getStdin()
+.then(input => {
+  let code;
+
+  if(input !== '') {
+    // run babel on the piped in data
+    code = babel.transform(input, {
+      extends: process.cwd() + '/' + '.babelrc'
```

Comment:
Less chars. HA. Lazy... Will swap for template string. Funny thing though now looking at this line theres concatenation for no reason and should probably use `path` instead. Will fix.